docs/pages/tabs.md

```diff
@@ -215,7 +215,7 @@ Add the attribute `data-deep-link="true"` to a tabstrip to:
 </div>
 ```
 
-For example, <a target="_blank" href="#panel3d">//example.com/#panel3d</a> will open the third tab panel at page load. This example will open a new browser tab and scroll you to the open tab.
+For example, <a target="_blank" href="#panel3d">http://example.com/#panel3d</a> will open the third tab panel at page load. This example will open a new browser tab and scroll you to the open tab.
```

Comment:
https://example.com works =)

Reply:
That does not work for me. My browser stats that its an invalid certificate.

Reply:
> hat does not work for me. My browser stats that its an invalid certificate.

Interesting, seem your root CA bundle is outdated. I will check that.

Reply:
Or maybe your system replaces this (MitM)? This should never be the case.

Reply:
<img width="599" alt="grafik" src="https://user-images.githubusercontent.com/827205/67621785-8e7af280-f813-11e9-8ba7-6759c598c5cf.png">

Please check what certificate your browser is showing. There might be something wrong with your system.

Reply:
@joeworkman

<img width="599" alt="grafik" src="https://user-images.githubusercontent.com/827205/67621785-8e7af280-f813-11e9-8ba7-6759c598c5cf.png">

Please check what certificate your browser is showing. There might be something wrong with your system.

Reply:
This is what I get... https://d.pr/i/6B6uTD

Reply:
Yes, a Zscaler setup replaces it with its own certificate. Some MitM there. This seems to come from your network where you are currently in. Is this the case?

Reply:
You are testing https://example.org. that works for me. https://example.com does not. I will move it all to example.org

Reply:
Nope. If I VPN out, it works for me to. Even .com. I am not sure what is doing that.

Reply:
I made them all https now as well

Reply:
This is because some Zscaler setup in your network has example.com in its rules.

Reply:
I tested .com, the CN is the main name from the cert, the alternative CN list contains all others.

Reply:
<img width="592" alt="grafik" src="https://user-images.githubusercontent.com/827205/67622174-f6333c80-f817-11e9-9fa2-92d3d4d69c4e.png">

Reply:
Please use `https://example.com` even if you have the problem - and let your system check why it is routed through a Zscaler proxy. This should never be the case and they should not replace the cert of example.com for security reasons.

That it is not trusted means that the root CA and the certificates of Zscaler was not added to your system CA bundle.

Either something is running on your system or on the network that does this.

If people getz the chance to detect such issues by using example.com, please let's use the `https://example.com`. MitM is not ok - and what currently happens in your case.

Reply:
I already changed it.

Reply:
I made the change as soon as I realized that the issue was just on my connection.